packages/wordcount/README.md

```diff
@@ -0,0 +1,17 @@
+# @wordpress/wordcount
+
+A utility to count words
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/wordcount@next --save
+```
+
+## Usage
+
+```JS
+import {WordCounter} from '@wordpress/wordcount';
+const count = WordCounter.count( 'Words to count', 'words', {});
```

Comment:
Minor: Maybe addding some documentation explaining the arguments and the important options we can pass.